lb/lb.go

```
@@ -1,23 +1,154 @@
+		lb.resolve(w, r)
+	})
+
+	return http.ListenAndServe(lb.port, nil)
+}
+
+func (lb *LoadBalancer) resolve(w http.ResponseWriter, req *http.Request) {
+	responseChan := make(chan *http.Response, 1) // buffered channel to avoid goroutine leak
+	lb.requestChan <- &RequestContext{request: req, responseChan: responseChan}
+	response := <-responseChan
+	if response != nil {
+		response.Write(w)
```

golangci-lint (1.62.2)

Error return value of `response.Write` is not checked

(errcheck)